.github/workflows/push_docs.yaml

```diff
@@ -134,6 +141,26 @@ jobs:
           echo "Changed files count: $CHANGED_FILES"
           echo "CHANGES=$CHANGED_FILES" >> $GITHUB_ENV
 
+      - name: Build the documentation
+        if: env.CHANGES != '0' && github.ref_name != env.MAIN_BRANCH
+        run: |
+          cd Tech-docs
+          sed -i "s#baseUrl: '\/'#baseUrl: '\/${{ env.TECH_DOCS_BRANCH }}'#" docusaurus.config.js
+          mkdir -p build
+          ./local-dev --build-only
```

Comment:
Shouldn't this command point to a specific repository? 
```
./local-dev --build-only <repo> 
```
Right now it builds only techdocs itself iiuic and the goal was to have an artifact of given remote repo.

Reply:
Nice pick, thanks